### PR TITLE
bert_extraction run_extraction_squad.sh error fixes

### PR DESCRIPTION
**Pull request by naveenjafer**
In continuation of the fixes in #57 . With this the entire run for extracting squad v1.1 is tested end to end with both CPU and GPU environments.

### CODE REVIEW

File: language/bert_extraction/steal_bert_qa/data_generation/preprocess_thief_dataset_squad.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
 import json
 import random
 
-from bert_extraction.steal_bert_qa.data_generation import preprocess_util as pp_util
+from language.bert_extraction.steal_bert_qa.data_generation import preprocess_util as pp_util
```
Comment (martiansideofthemoon) [inline language/bert_extraction/steal_bert_qa/data_generation/preprocess_thief_dataset_squad.py:19]: @kentonl could you confirm this? I vaguely remember this was correct in November, but have things changed now?

Comment (naveenjafer) [reply language/bert_extraction/steal_bert_qa/data_generation/preprocess_thief_dataset_squad.py:19]: When I did run this without the language module name prepended it ran into errors telling that bert_extraction doest exist etc. Maybe there is a difference in the way we installed the language package? At the top most level of language repo, I build it using the command
`python3 setup.py build && python3 setup.py install`

Comment (martiansideofthemoon) [reply language/bert_extraction/steal_bert_qa/data_generation/preprocess_thief_dataset_squad.py:19]: Seems okay, I did find the `from language.` prefix in other places of the repository. @kentonl does this break things when the code is compiled internally?

**Comment by kentonl** [comment]
Thanks, merged!

**Pull request closed without merging**